parsers/IN_WE.py

```diff
@@ -57,6 +58,12 @@
     "consumption": {"url": CONSUMPTION_URL, "datetime_column": "current_datetime"},
 }
 
+get_date_range = lambda dt: pd.date_range(
```

Comment:
I'm not a super fan of using lambda functions like this, we're using it 3 times in this module, I think a regular function definition would be just fine. It would add more consistency and if we hover over the function we would get type hints and definition.

So something like this:
```
def get_date_range(dt: pd.DateRange) -> PyDatetime:
your magic
```